### DISCUSSION TITLE
Website Preview on VScode Server

### DISCUSSION THREAD

**Discussion by berk-efe** [gitpod-io/openvscode-server#415] [Polls]
Hello, is there a way i can preview my website (website is inside my aws ec2 instance)

I tried to change ip of live server but did not work.

![resim](https://user-images.githubusercontent.com/92454444/184960070-cc0c7687-0228-4fd2-a6f0-42874c797e8b.png)

i wrote to "Live Server > Settings: Host" my ec2 instance's public ip

but i got timeout

**Comment by jeanp413** [comment]
@AkioKiyota for preview extension or any other extension that needs some service in a different port:
- Ideal fix, you need to configure your own reverse proxy, so different origins/subdomains are mapped to a port in the remote host (this is what gitpod/codespaces do) e.g. 
      - `https://gitpodio-templatetypesc-sskm55x3tyr.ws-us47.gitpod.io/` (this is mapped to openvscode-server port) 
      -  `https://3000-gitpodio-templatetypesc-sskm55x3tyr.ws-us47.gitpod.io` this is mapped to port `3000` in the remote host where an http server is running
- Create an ssh tunnel to the port your service is running e.g. `ssh -L 9000:192.168.0.1:9000 user@192.168.0.1` and then access it using `http://localhost:9000`

**Reply by berk-efe** [reply]
thank you a lot i will give it a try
